<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CardWidget" name="MainCard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1061</width>
     <height>351</height>
    </rect>
   </property>
   <widget class="SubtitleLabel" name="SubtitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine details: </string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1041</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ElevatedCardWidget" name="ElevatedCardWidget_7">
         <widget class="BodyLabel" name="MachineName">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="ElevatedCardWidget" name="ElevatedCardWidget_6">
         <widget class="BodyLabel" name="MachineIP">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget_5">
       <widget class="BodyLabel" name="BodyLabel_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>63</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Uptime:</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="SubtitleLabel" name="Uptime">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget_11">
       <widget class="BodyLabel" name="BodyLabel_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Total Disk:</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item alignment="Qt::AlignHCenter">
          <widget class="SubtitleLabel" name="TotalDisk">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget_10">
       <widget class="BodyLabel" name="BodyLabel_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Cpu cores:</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item alignment="Qt::AlignHCenter">
          <widget class="SubtitleLabel" name="CpuCores">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget_9">
       <widget class="BodyLabel" name="BodyLabel_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Total Ram:</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item alignment="Qt::AlignHCenter">
          <widget class="SubtitleLabel" name="TotalRam">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget_8">
       <widget class="BodyLabel" name="BodyLabel_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>NICs:</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item alignment="Qt::AlignHCenter">
          <widget class="SubtitleLabel" name="Nics">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1041</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget">
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-5</y>
          <width>341</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="ProgressRing" name="CPUusagering">
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="ProgressRing" name="Diskusagering">
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ProgressRing" name="RAMusagering">
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel">
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
           <property name="text">
            <string>RAM</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_3">
           <property name="text">
            <string>Disk</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ElevatedCardWidget" name="ElevatedCardWidget_13">
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>341</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="ProgressRing" name="min1LoadRing">
             <property name="textVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter">
            <widget class="StrongBodyLabel" name="StrongBodyLabel_4">
             <property name="text">
              <string>1 Min load</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="ProgressRing" name="min15LoadRing">
             <property name="textVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2" alignment="Qt::AlignHCenter">
            <widget class="StrongBodyLabel" name="StrongBodyLabel_6">
             <property name="text">
              <string>15 Min load</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ProgressRing" name="min5LoadRing">
             <property name="textVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignHCenter">
            <widget class="StrongBodyLabel" name="StrongBodyLabel_5">
             <property name="text">
              <string>5 Min load</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ElevatedCardWidget" name="ElevatedCardWidget_2">
       <widget class="QWidget" name="gridLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="1">
          <widget class="BodyLabel" name="TotalCachedMemory">
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_11">
           <property name="text">
            <string>Total cached Memory:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_9">
           <property name="text">
            <string>Total swap:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="BodyLabel" name="TotalSwap">
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="BodyLabel" name="AvailableSwap">
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_10">
           <property name="text">
            <string>Available swap:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="HorizontalSeparator" name="HorizontalSeparator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>321</width>
          <height>3</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
           <property name="text">
            <string>Network Out:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="BodyLabel" name="NetworkOut">
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="BodyLabel" name="NetworkIn">
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="StrongBodyLabel" name="StrongBodyLabel_8">
           <property name="text">
            <string>Network In:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CaptionLabel" name="TimestampLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>   As of Null</string>
    </property>
   </widget>
  </widget>
  <widget class="CardWidget" name="CardWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>1061</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="PrimaryPushButton" name="PrimaryPushButton">
        <property name="text">
         <string>Refresh Graphs</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="LastHourTab">
       <attribute name="title">
        <string>Last Hour</string>
       </attribute>
       <widget class="QTabWidget" name="TabLastHour">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1041</width>
          <height>371</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="CPULastHour">
         <attribute name="title">
          <string>CPU, RAM, Disk usage</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="LoadsLastHour">
         <attribute name="title">
          <string>1,5,15 mins Loads</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="NetworkLastHour">
         <attribute name="title">
          <string>Network Usage</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Last24HoursTab">
       <attribute name="title">
        <string>Last 24 Hours</string>
       </attribute>
       <widget class="QTabWidget" name="TabLast24Hours">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1041</width>
          <height>371</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="CPULast24Hours">
         <attribute name="title">
          <string>CPU, RAM, Disk usage</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="LoadsLast24Hours">
         <attribute name="title">
          <string>1,5,15 mins Loads</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="NetworkLast24Hours">
         <attribute name="title">
          <string>Network Usage</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="CustomTab">
       <attribute name="title">
        <string>Custom</string>
       </attribute>
       <widget class="QToolBox" name="toolBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1041</width>
          <height>361</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Intervalselection">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1041</width>
           <height>307</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Interval selection</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1041</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="TimePicker" name="StartTime"/>
             </item>
             <item row="1" column="1">
              <widget class="CalendarPicker" name="EndDate">
               <property name="text">
                <string>End Date</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="CalendarPicker" name="StartDate">
               <property name="text">
                <string>Start Date</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="TimePicker" name="EndTime"/>
             </item>
             <item row="1" column="3">
              <widget class="PrimaryPushButton" name="PlotButton">
               <property name="text">
                <string>Plot</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="LineEdit" name="Ticks">
               <property name="placeholderText">
                <string>Ticks</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Plots">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1041</width>
           <height>307</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Plots</string>
         </attribute>
         <widget class="QTabWidget" name="CustomSubTab">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1041</width>
            <height>311</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="CPUCustom">
           <attribute name="title">
            <string>CPU, RAM, Disk usage</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="LoadsCustom">
           <attribute name="title">
            <string>1,5,15 mins Loads</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="NetworkCustom">
           <attribute name="title">
            <string>Network Usage</string>
           </attribute>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElevatedCardWidget</class>
   <extends>SimpleCardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarPicker</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TimePicker</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressRing</class>
   <extends>ProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>